app/models/eligibility_factory.rb

```diff
@@ -1,12 +1,12 @@
 class EligibilityFactory
-  attr_reader :auction
+  attr_reader :start_price_threshold
 
   def initialize(auction)
-    @auction = AuctionPresenter.new(auction)
+    @start_price_threshold = StartPriceThresholds.new(auction.start_price)
```

Comment:
This will eventually be `AuctionThresholds` when https://github.com/18F/micropurchase/pull/617 is merged. The constructor will take an `Auction` instance as the sole argument.

Just some notes for the impending rebase!